src/pages/dashboard/dashboard.ts

```diff
@@ -15,18 +16,19 @@ export class DashboardPage {
   title: string = 'Dashboard';
 
   constructor(public navCtrl: NavController, public navParams: NavParams) {
+    navCtrl.remove(0);
   }
 
   ionViewDidLoad() {
     console.log('ionViewDidLoad DashboardPage');
   }
 
   goToPage(page: string) {
-    if (page === 'JournalPage') return this.navCtrl.push(JournalPage);
-    if (page === 'LdtmModePage') return this.navCtrl.push(LdtmModePage);
-    if (page === 'PolicyDataPage') return this.navCtrl.push(PolicyDataPage);
-    if (page === 'ManageDeviceDebriefStoragePage') return this.navCtrl.push(ManageDeviceDebriefStoragePage);
-    if (page === 'TrainerModePage') return this.navCtrl.push(TrainerModePage);
+    if (page === DashboardComponentLinks.JournalPage) return this.navCtrl.push(JournalPage);
```

Comment:
looks much better, I would change these series of if statements to switch and it would be super